### PR TITLE
Prepare release

### LINKED ISSUE #139: Prepare version 2.0
I want to release 2.0 in September. At the moment there is no open issues or anything stopping us. We just need to wait for a final release of PSR-18. 

@joelwurtz made [a comment](https://github.com/php-http/httplug/pull/128/files/cbc903aadbe53e2dd39ebc700e6a5aea00b2bc46#r204876337) about async. Is that something we should do before 2.0?

### PR DESCRIPTION
**Pull request by sagikazarmark**
Closes #139

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +13,15 @@
 **HTTPlug, the HTTP client abstraction for PHP.**
 
 
+## Intro
+
+HTTPlug served as a reference is the predecessor of [PSR-18](https://github.com/php-fig/fig-standards/blob/master/accepted/http-client/http-client.md)
```
Comment (dbu) [inline README.md]: s/served as a reference//

File: README.md [hunk 2]
```diff
@@ -22,14 +31,6 @@ $ composer require php-http/httplug
 ```
 
 
-## Intro
-
-This is the contract package for HTTP Client.
-Use it to create HTTP Clients which are interoperable and compatible with [PSR-7](http://www.php-fig.org/psr/psr-7/).
```
Comment (dbu) [inline README.md:28]: i'd love to keep mentioning PSR-7 somewhere in the intro

File: composer.json [hunk 3]
```diff
@@ -32,10 +32,5 @@
     "scripts": {
         "test": "vendor/bin/phpspec run",
         "test-ci": "vendor/bin/phpspec run -c phpspec.ci.yml"
-    },
-    "extra": {
-        "branch-alias": {
-            "dev-2.x": "2.0.x-dev"
```
Comment (dbu) [inline composer.json:38]: we should alias master to 2.x-dev when this is merged.

File: README.md [hunk 4]
```diff
@@ -13,6 +13,15 @@
 **HTTPlug, the HTTP client abstraction for PHP.**
 
 
+## Intro
+
+HTTPlug served as a reference is the predecessor of [PSR-18](https://github.com/php-fig/fig-standards/blob/master/accepted/http-client/http-client.md)
+HTTP Client standard. Since there is an entire ecosystem built around HTTPlug and since it's already widely adapted,
+this package stays available, but new implementations and consumers should use the PSR-18 interfaces.
```
Comment (dbu) [inline README.md]: could we say instead "we will keep maintaining this package for the time being, but new ..."

File: README.md (hunk 1 above)
Comment (dbu) [inline README.md]: and we should mention that version 2 of httplug extends the psr-18 interfaces to allow a convenient migration path.

**Review by dbu — COMMENTED** [review]
thanks. commented on some details.

**Comment by sagikazarmark** [comment]
Thanks for the review @dbu, I made the requested changes.

File: README.md [hunk 5]
```diff
@@ -13,6 +13,19 @@
 **HTTPlug, the HTTP client abstraction for PHP.**
 
 
+## Intro
+
+HTTPlug is the predecessor of [PSR-18](http://www.php-fig.org/psr/psr-18/)
+HTTP Client standard built on [PSR-7](http://www.php-fig.org/psr/psr-7/) HTTP messages.
+Since there is an entire ecosystem built around HTTPlug which is already widely adapted,
```
Comment (dbu) [inline README.md]: s/adapted/adopted/

File: README.md [hunk 6]
```diff
@@ -13,6 +13,19 @@
...
+we will keep maintaining this package for the time being,
+but new implementations and consumers should use the PSR-18 interfaces.
+HTTPlug 2.x extends the PSR-18 interfaces to allow a convenient migration path,
+but in time it may be completely replaced by the standard.
```
Comment (dbu) [inline README.md]: i would put `.`on the line above and then say "In the long term, we expect PSR-18 to completely replace the need for Httplug."

**Review by dbu — COMMENTED** [review]
sorry, some further nitpicks.

**Review by dbu — APPROVED** [review]
thanks!

**Comment by dbu** [comment]
now we need to wait for psr-18 to be tagged 1.0.0, i guess?

**Comment by sagikazarmark** [comment]
Yeah. I think @Nyholm has access to it from our side, but I don't think there are any blockers there.

**Review by Nyholm — APPROVED** [review]
Excellent

**Pull request merged**